src/streamlink/plugins/twitch.py

```diff
@@ -42,19 +34,19 @@
 TWITCH_CLIENT_ID = "pwkzresl8kj2rdj6g7bvxl9ys1wly3j"
 
 _url_re = re.compile(r"""
-    http(s)?://
+    https?://
     (?:
         (?P<subdomain>[\w\-]+)
         \.
     )?
-    twitch.tv/
+    twitch\.tv/
     (?:
         videos/(?P<videos_id>\d+)|
         (?P<channel>[^/]+)
     )
     (?:
         /
-        (?P<video_type>[bcv])(?:ideo)?
+        v(?:ideo)?
```

Comment:
Do these URLs even exist anymore? I'm getting a 404. The forward-slash above is also wrong here. I think the `video_id` group can be removed in favor of `videos_id`, although not 100% sure.

Reply:
most of it is gone,

`https://www.twitch.tv/twitch/v/292713971` just redirects to
`https://www.twitch.tv/videos/292713971`

Reply:
The first URL doesn't get matched by this group

Reply:
It doesn't need to if it redirects? It might break some old URLs people have around if they have the redirect option disabled...